ra_aid/tools/programmer.py

```
@@ -149,5 +190,7 @@ def parse_aider_flags(aider_flags: str) -> List[str]:
     return [f"--{flag.lstrip('-')}" for flag in flags if flag.strip()]
 
 
+
+
 # Export the functions
-__all__ = ["run_programming_task"]
+__all__ = ["run_programming_task", "get_aider_executable"]
```

Ruff (0.8.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)